modules/loader.py

```diff
@@ -6,6 +6,21 @@ def __init__(self, bot):
         self.version = "1.0.1"
         self.bot = bot
 
+    async def cog_command_error(self, ctx, error):
+        if isinstance(error, commands.errors.CommandInvokeError):
+            print(f"[WARN] {str(error)}")
+            if isinstance(error.original, commands.errors.ExtensionNotLoaded):
+                return await ctx.send("[!] Module not already loaded.")
+            elif isinstance(error.original, commands.errors.ExtensionAlreadyLoaded):
+                return await ctx.send("[!] Module already loaded.")
+            elif isinstance(error.original, commands.errors.ExtensionNotFound):
+                print("YEEET")
```

Comment:
Please remove this. (It's funny though)

Reply:
Whoops that was debug I forgot to remove lol